src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -405,6 +405,12 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
                                 await _callbacks.Plot(message.GetString(0, "xaml_file_path"), ct);
                                 break;
 
+                            case "Locator":
```

Comment:
It's not an R function - it comes from the graphics device.

https://github.com/Microsoft/R-Host/pull/63